README.md

```diff
@@ -43,6 +43,7 @@ A network request is made to the MapQuest `RouteService` with these coordinates
 
 One or more alternate `Routes` are returned from the `RouteService`, and these are plotted on the `MapView`. The user chooses one desired `Route` to navigate -- this is passed along to the `NavigationActivity` to start navigation (using the `NavigationManager`).
 
+Before a route is passed to the `NavigationActivity` we present a UI Dialog to get the users consent for traffic data collection. This value is then used to call `NavigationManager.setUserLocationTrackingConsentStatus()` with the correct `UserLocationTrackingConsentStatus` value. Setting this value is required before a route can be started.
```

Comment:
Channelling my inner Daniel here.... Please don't hate me.

Perhaps replace ...
"_users consent for traffic data collection. This value is then used to call `NavigationManager.setUserLocationTrackingConsentStatus()` with the correct `UserLocationTrackingConsentStatus` value._"

with

"_user's consent for traffic data collection. Their preference is to be set by calling `NavigationManager.setUserLocationTrackingConsentStatus()` with the corresponding `UserLocationTrackingConsentStatus` value._"

Reply:
Haha! I like your suggestion. Fixed